### PR TITLE
Integrate capstone with oss-fuzz

### PR DESCRIPTION
**Pull request by catenacyber**
@aquynh as you asked, this is the same as #1145 but for master branch

### CODE REVIEW

**Comment by catenacyber** [comment]
I am wondering why I got a green build with #1145 and not here... working on it

**Comment by aquynh** [comment]
merged, thanks!

**Comment by aquynh** [comment]
i cherry-picked this for the next branch, too. but next branch has few more architectures, please updating that.

**Comment by aquynh** [comment]
could you send another pull req to add your name to https://github.com/aquynh/capstone/blob/master/CREDITS.TXT?

**Comment by catenacyber** [comment]
> i cherry-picked this for the next branch, too. but next branch has few more architectures, please updating that.

yes for the next branch, this is #1145 
Is that one ok for you ?

**Comment by aquynh** [comment]
this breaks Appveyor, would you take a look: https://ci.appveyor.com/project/aquynh/capstone/build/4.0-943?

**Comment by aquynh** [comment]
regarding #1145, EVM arch is still missing?

**Comment by catenacyber** [comment]
Here is a new pull request to fix appveyor #1152

File: suite/fuzz/onefile.c [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    Size = ftell(fp);
+    if (Size == (size_t) -1) {
+        fclose(fp);
+        return 2;
+    }
+    if (fseek(fp, 0L, SEEK_SET) != 0) {
+        fclose(fp);
+        return 2;
+    }
+    Data = malloc(Size);
```
Comment (kcc) [inline suite/fuzz/onefile.c]: This malloc is leaked, isn't it? 
This means that we can't use this stub with lsan.

Comment (catenacyber) [reply suite/fuzz/onefile.c]: Indeed, so I am fixing it :-) (see pull request)

**Comment by catenacyber** [comment]
@kcc leak is fixed :-)

**Pull request merged**